src/Tribe/Tickets_Handler.php

```diff
@@ -93,9 +98,10 @@ public function __construct() {
 		foreach ( $main->post_types() as $post_type ) {
 			add_action( 'save_post_' . $post_type, array( $this, 'save_image_header' ) );
 			add_action( 'save_post_' . $post_type, array( $this, 'save_global_stock' ) );
-			add_action( 'save_post_' . $post_type, array( $this, 'save_tickets_order' ) );
 		}
 
+		add_action( 'save_post_tribe_events', array( $this, 'save_tickets_order' ) );
```

Comment:
prior to this change the `save_tickets_order` method was hooked to `save_post_{post_type}` [line `96`]

 is it **intentional** to unhook this method from all `save_post_{post_type}` hooks except `tribe_events`?